src-tauri/Cargo.toml

```diff
@@ -77,34 +111,46 @@ struct_iterable = "0.1.1"
 regex = "1.11.1"
 shlex = "1.3.0"
 zstd = "0.13.3"
-chrono = { version = "0.4.42", default-features = false, features = ["serde", "clock"] }
+chrono = { version = "0.4.42", default-features = false, features = [
+    "serde",
+    "clock",
+] }
 once_cell = "1.21.3"
-idf-env = { git = "https://github.com/espressif/idf-env", rev="fd69ab4f550ef35647bb32d1584caa6623cbfc4e" }
+idf-env = { git = "https://github.com/espressif/idf-env", rev = "fd69ab4f550ef35647bb32d1584caa6623cbfc4e" }
 fs_extra = { version = "1.3.0", optional = true }
 
 # GUI-related dependencies (optional)
 tauri = { version = "2.7.0", features = [], optional = true }
 tauri-plugin-shell = { version = "2.3.0", optional = true }
 tauri-plugin-dialog = { version = "2.3.2", optional = true }
 tauri-plugin-log = { version = "2.6.0", optional = true }
-num_cpus =  {version = "1.17.0", optional = true }
-tauri-plugin-store = { version="2.0.0", optional = true }
-tauri-plugin-opener = { version="2.5.0", optional = true }
+num_cpus = { version = "1.17.0", optional = true }
+tauri-plugin-store = { version = "2.0.0", optional = true }
+tauri-plugin-opener = { version = "2.5.0", optional = true }
 
 # CLI-related dependencies
-clap = {version = "4.5", features = ["cargo", "derive", "color"], optional = true}
-dialoguer = { git = "https://github.com/Hahihula/dialoguer.git", branch = "folder-select", features = ["folder-select"], optional = true }
+clap = { version = "4.5", features = [
+    "cargo",
+    "derive",
+    "color",
+], optional = true }
+dialoguer = { git = "https://github.com/Hahihula/dialoguer.git", branch = "folder-select", features = [
+    "folder-select",
+], optional = true }
 indicatif = { version = "0.17.8", optional = true }
 console = { version = "0.15.8", optional = true }
 log4rs = { version = "1.3.0", optional = true }
 
 
 # userustpython feature dependencies
-rustpython-vm = { git = "https://github.com/Hahihula/RustPython.git", branch = "test-rust-build", features = ["freeze-stdlib"], optional = true }
-rustpython-stdlib = { git = "https://github.com/Hahihula/RustPython.git", branch = "test-rust-build", features = ["ssl-vendor"], optional = true }
+rustpython-vm = { git = "https://github.com/Hahihula/RustPython.git", branch = "test-rust-build", features = [
+    "freeze-stdlib",
+], optional = true }
+rustpython-stdlib = { git = "https://github.com/Hahihula/RustPython.git", branch = "test-rust-build", features = [
+    "ssl-vendor",
+], optional = true }
 os_info = "3.12.0"
-
-
+clap_complete = "4.5.58"
```

Comment:
nitpick but could you add this to the cli dependencies and make it optional and install only if cli feature is enabled?